docs/README.md

```diff
@@ -16,6 +16,8 @@ The script will generate all require changes based on your input, please review,
 
 If you are the project owner, you can set up the [Publish to BCR](https://github.com/apps/publish-to-bcr) Github App for your repository to automatically send a PR to the BCR when cutting a new release.
 
+When manually editing files you may find `bazel run -- //tools:update_integrity foomod` useful to calculate the integrity hashes in foomod's sources.json file.
+
```

Comment:
Could you also mention that this uses the latest version by default and that `--version` can be used to override it?

Reply:
```suggestion
The tool also accepts a `--version` option to update the source.json of a specific version of the module (instead of latest).

```